### ISSUE TITLE
NullPointerException being thrown in SmoochService

### ISSUE THREAD

**Issue by hitherejoe** [smooch/smooch-android#111]
We're getting this exception thrown on a range of Android versions (4 to 8) and also a range of devices.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'io.smooch.core.c.aa io.smooch.core.c.u.e()' on a null object reference
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService$10$1.a(Unknown Source)
       at io.smooch.core.service.SmoochService$10$1.a(Unknown Source)
       at io.smooch.core.b.f$13.a(Unknown Source)
       at io.smooch.core.b.a.b$1$2.run(Unknown Source)
       at android.os.Handler.handleCallback(Handler.java:743)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:150)
       at android.app.ActivityThread.main(ActivityThread.java:5621)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
```

I saw some similar NPE stack traces in the issues, but they look a little different to this. Any idea what could be going on here? 🤔

**Comment by wmora** [comment]
which SDK version are you using?

**Comment by hitherejoe** [comment]
My bad @wmora - we're using 5.7.3, I noticed this was marked as fixed in 5.7.1. WIll close this for now 👍
